wpa_supplicant/wpa_supplicant.c

```diff
@@ -8229,9 +8229,11 @@ void dump_freq_data(struct wpa_supplicant *wpa_s, const char *title,
 	wpa_dbg(wpa_s, MSG_DEBUG, "Shared frequencies (len=%u): %s",
 		len, title);
 	for (i = 0; i < len; i++) {
+#if CONFIG_WIFI_NM_WPA_SUPPLICANT_DEBUG_LEVEL <= MSG_DEBUG
```

Comment:
Hmm, the config option is only available in zephyr, isn't there a config option that is set also in supplicant upstream code and in zephyr for debug level?

Reply:
Or could we do similar things as what `ARG_UNUSED()` does?

Reply:
> isn't there a config option that is set also in supplicant upstream code and 

No, and even if it did the code doesn't get compiled out, so, this warning won't be seen. This was an memory optimization done by Nordic.